src/components/Modals/FiatRamps/views/Overview.tsx

```diff
@@ -121,17 +139,76 @@ export const Overview: React.FC<OverviewProps> = ({
   chainId,
   setChainId,
   chainAdapterManager,
+  handleAccountIdChange,
+  accountId,
 }) => {
   const translate = useTranslate()
   const { fiatRampAction } = useParams<{ fiatRampAction: FiatRampAction }>()
   const toast = useToast()
   const { fiatRamps } = useModal()
-
-  const [shownOnDisplay, setShownOnDisplay] = useState<Boolean | null>(null)
-
   const {
     state: { wallet },
   } = useWallet()
+  const multiAccountsEnabled = useFeatureFlag('MultiAccounts')
+  const [shownOnDisplay, setShownOnDisplay] = useState<Boolean | null>(null)
+  const accountBalances = useAppSelector(selectPortfolioAccountBalances)
+  const marketData = useAppSelector(state =>
+    selectMarketDataById(state, selectedAsset?.assetId ?? ''),
+  )
+  const [accountAddress, setAccountAddress] = useState<string | null>(null)
+
+  // TODO: change the following with `selectPortfolioAccountMetadataByAccountId`
+  // once web/#2632 got merged
+  const accountsMeta = useAppSelector(selectPortfolioAccountMetadata)
```

Comment:
Don't pluralize (this *does* refer to `AccountMetadataById` i.e a record of AccountMetdata, however, we should use a consistent naming with selectors), and use `accountMetadata` for consistency
```suggestion
  const accountMetadata = useAppSelector(selectPortfolioAccountMetadata)
```